source/tutorials/nix-language.md

```diff
@@ -1414,22 +1412,117 @@ Parentheses are required to separate function declaration from function applicat
 
 </details>
 
+(pkgs-lib)=
 ### `pkgs.lib`
 
-The [`nixpkgs`][nixpkgs] repository contains an attribute set called [`lib`][nixpkgs-lib], which provides a large number of useful functions.
+The [`nixpkgs`][nixpkgs] repository contains an attribute set called [`lib`][nixpkgs-lib], which provides a large number of useful functions implemented in the Nix language.
```

Comment:
I'm not sure if this adds clarity: it looks like this may be interpreted either as "they are implemented using the language" or "they are part of the language"... I think if we don't add anything here it's pretty clear that builtins are the exception and these are regular nix functions that were just explained.

Reply:
That change was to address [this comment](https://github.com/NixOS/nix.dev/pull/267#issuecomment-1270076332). I didn't want to make a separate sentence, but probably it would be sensible here.

```suggestion
The [`nixpkgs`][nixpkgs] repository contains an attribute set called [`lib`][nixpkgs-lib], which provides a large number of useful functions.
They are implemented in the Nix language.
```

Reply:
How about "They are implemented in the Nix language", to get rid of that potential confusion ?